node/12/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:12.13.0-slim
+
+RUN apt-get update -y && \
+      apt-get install -y \
+        jq \
+        nano \
+        htop && \
+      apt-get clean && \
+      rm -rf /var/lib/apt/lists/*
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app 
```

Comment:
Please remove the extra space at the end of this line. Depending on your editor, you may want to enable that automatically. Here's how to do that in Sublime Text 3:

```json5
    // Set to true to removing trailing white space on save
    "trim_trailing_white_space_on_save": true,
```

Reply:
No.  I will need to stay conscious of it for now.  My editor has been acting up with that setting on.  Thank you for advice though, would normally agree with it.  Gonna look at using different editor soon anyways.

Reply:
>Gonna look at using different editor soon anyways.
You know... Vim is plain awesome! \o/